site/docs/components/progress-bar.mdx

```diff
@@ -10,12 +10,13 @@ needToKnow:
 health:
   designed: true
   documented: true
-  implemented: false
-  latestDesign: false
-  allVariants: false
+  implemented: true
```

Comment:
It's worth noting that the Figma file does not have a design for the "indeterminate" variant mentioned in  https://cultureamp.design/components/progress-bar/ so that has not been implemented.